### ISSUE TITLE
No error message if unable to download files

### ISSUE THREAD

**Issue by crkellen** [SrKomodo/shadowfox-updater#44]
Using the v2.0.0 updater for Windows 10 64bit, on my work computer, which blocks all domains not explicitly allowed, the updater finds the profiles, asks all the relevant prompts, creates the internal UUID's, but doesn't create any userChrome.css or userContent.css files.

I am assuming this is due to it not being downloaded, which I worked around by manually downloading the files from github (which I had to do via copy/paste, as raw.github is blocked), and modifying them appropriately.

It would be nice to have a backup system in place, where, if not connected to the internet, or unable to download, the user is prompted to use the built in files, which may be outdated, if your installer is outdated.

I'm not sure how feasible this is, but I think it would be useful for situations like mine, or worse than mine (since I was still able to access github, I could still manually copy paste).

**Comment by NomDeMorte** [comment]
Meanwhile, call your helpdesk and have them unblock this stuff which is an Occupational Health and Safety requirement in order to minimise eye strain. If you ask, they basically HAVE to do it for you - it's the law. Source: I was a helpdesk boss and OHS trained.

**Comment by crkellen** [comment]
What specifically needs to be unblocked? What does it try to pull down/access?

**Comment by SrKomodo** [comment]
The only thing it needs is access to `raw.githubusercontent.com`, although the program should definitely show an error message instead of silently failing.

**Comment by arguablykomodo** [comment]
I remade the whole error handling system for v2.2.0, which should hopefully fix this issue. Do not hesitate to reopen if the issue persists though.
